File: YoutubePlay.Model/Youtube/YoutubeVideo.cs
namespace YoutubePlay.Models.Youtube
{
    public class YoutubeVideo
    {
        public string Title { get; set; }

        public Thumbnails Thumbnails { get; set; }

        public string Duration { get; set; }

        public string TimeAgo { get; set; }

        public ChannelInfo Channel { get; set; }
        public string ViewCount { get; set; }
        public string ShortViewCount { get; set; }
        public string Id { get; set; }
    }
}

File: YoutubePlay.UI/ClientApp/src/app/models/youtube-video.ts
import { BaseModel } from "./baseModel";
import { ChannelInfo } from "./channelInfo";
import { Thumbnails } from "./thumbnails";

export class YoutubeVideo extends BaseModel {

    title: string;
    thumbnails: Thumbnails;
    duration: string;
    timeAgo: string;
    channel: ChannelInfo;
    viewCount: string;
    shortViewCount: string;
    id: string;
}
